longevity_operator_multi_tenant_test.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2022 ScyllaDB
+
+import copy
+
+from longevity_test import LongevityTest
+
+from sdcm.utils.common import ParallelObject
+
+
+# pylint: disable=too-many-instance-attributes
+class ScyllaClusterStats(LongevityTest):
+    # pylint: disable=too-many-arguments,super-init-not-called
+    def __init__(self, db_cluster, loaders, monitors, prometheus_db, params, test_config, cluster_index):
+        self.db_cluster = db_cluster
+        self.loaders = loaders
+        self.monitors = monitors
+        self.prometheus_db = prometheus_db
+        self.params = copy.deepcopy(params)
+        self._es_doc_type = "test_stats"
+        self._stats = self._init_stats()
+        self.test_config = test_config
+        self._duration = self.params.get(key='test_duration')
+        self.create_stats = self.params.get(key='store_perf_results')
+        self.status = "RUNNING"
+        self.cluster_index = str(cluster_index)
+        self._test_id = self.test_config.test_id() + f"--{cluster_index}"
+        self._test_index = self.get_str_index()
+        self.create_test_stats()
```

Comment:
Same here? Do we need to create and save stats running longevity?

Reply:
if `create_stats` is False, it's a noop

in general I think we should try to create a way to share this class definition between tests, and not implemented almost the exact one, with those small changes, for each test that we would like to support with multi-tenancy

Reply:
> if `create_stats` is False, it's a noop
> 
> in general I think we should try to create a way to share this class definition between tests, and not implemented almost the exact one, with those small changes, for each test that we would like to support with multi-tenancy

This is what I wanted to do from beginning, but the classes inherite from different tests

Reply:
> Same here? Do we need to create and save stats running longevity?

removed